### ISSUE TITLE
How do I list all pages of licensees using the RESTful API?

### ISSUE THREAD

**Issue by kevinhiltb9** [Labs64/NetLicensing-Community#43]
**Question**
Using the RESTful API to [list licensees](https://netlicensing.io/wiki/licensee-services#licensees-list), how do I get all entries? I can only get the first page, and there are four total pages.

**Request URL**
`https://go.netlicensing.io/core/v2/rest/licensee`

**Response**
```
<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:netlicensing xmlns="http://www.w3.org/2000/09/xmldsig#" xmlns:ns2="http://netlicensing.labs64.com/schema/context">
    <ns2:infos/>
    <ns2:items pagenumber="0" itemsnumber="100" totalpages="4" totalitems="398" hasnext="true">
. . .
```

**Comment by r-brown** [comment]
Hello @kevinhiltb9
You can retrieve the next available pages by adding a filter to the [Licensee.list()](https://netlicensing.io/wiki/licensee-services#licensees-list) method

```
GET https://go.netlicensing.io/core/v2/rest/licensee?filter=page=1;items=100
```

---

related issue: https://github.com/Labs64/NetLicensing-Community/issues/27#issuecomment-773934112

**Comment by kevinhiltb9** [comment]
Thank you, I'll give it a shot.

Out of curiosity, is this mentioned anywhere in the documentation? I looked before opening an issue here and didn't see anything in the [wiki article](https://netlicensing.io/wiki/licensee-services#licensees-list). I did see the filter option in [JavaDoc for the list function](https://go.netlicensing.io/javadoc/v2/com/labs64/netlicensing/service/LicenseeService.html#list(com.labs64.netlicensing.domain.vo.Context,java.lang.String)), but it says "filter - reserved for the future use, must be omitted / set to NULL".
